src/test/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/FilterClauseDeserializerTest.java

```diff
@@ -429,6 +429,49 @@ public void mustHandleSize() throws Exception {
           .isEqualTo(expectedResult.getPath());
     }
 
+    @Test
+    public void mustHandleIntegerWithTrailingZeroSize() throws Exception {
+      String json = """
+          {"sizePath" : {"$size": 0.0}}
```

Comment:
Is this actually required to be legal? (curious, maybe Mongo allows it?)

Reply:
I checked Mongo Side. They allow it.